### PR TITLE
Colorize logs

### PR DESCRIPTION
**Pull request by bragagia**

### CODE REVIEW

File: apps/logs.go [hunk 1]
```diff
@@ -84,7 +84,10 @@ func dumpLogs(logsURL string, n int, filter string) error {
 		return nil
 	}
 
-	_, err = stdio.Copy(os.Stdout, res.Body)
+	//_, err = stdio.Copy(os.Stdout, res.Body)
+	buf := new(bytes.Buffer)
```
Comment (Soulou) [inline apps/logs.go]: With this method, you're putting everything in one big buffer, imagine there are A LOT of logs. That's something you want to do line by line with a bytes.StringReader for instance.

**Comment by Soulou** [comment]
It would be great to have special coloring for the router logs

```
2017-05-29 12:14:48.099387055 +0200 CEST [router] method=GET path="/v1/provider" host=<host> request_id=0059d1b6-d92c-413f-b1c6-cd8e1dfeecdc from="167.114.245.201" protocol=https status=404 duration=0.007s bytes=544 referer="-" user_agent="Go-http-client/1.1"
```

**Comment by bragagia** [comment]
#288

**Comment by bragagia** [comment]
Router log look likes that :
2017-05-29-164941_3808x2075_scrot

**Comment by bragagia** [comment]
image

**Pull request merged**